Mage.Sets/src/mage/cards/e/EnthrallingHold.java

```diff
@@ -62,3 +54,16 @@ public EnthrallingHold copy() {
         return new EnthrallingHold(this);
     }
 }
+
+class EnthrallingHoldTarget extends TargetCreaturePermanent {
```

Comment:
Missing override copy constructor. Original card (DreamLeash) must be fixed too.

Reply:
fixed